eventmesh-runtime/conf/eventmesh.properties

```diff
@@ -74,7 +74,7 @@ eventMesh.server.registry.enabled=false
 eventMesh.metrics.prometheus.port=19090
 
 #trace exporter
-eventmesh.trace.exporter.type=log
+eventmesh.trace.exporter.type=zipkin
```

Comment:
here may be Zipkin, because of your class name

Reply:
changed